composer.json

```diff
@@ -19,12 +49,16 @@
         "drupal/block_blacklist": "~1.0",
         "drupal/chosen": "~2.4",
         "drupal/components": "~1.0",
+        "drupal/config_distro": "^1.0-alpha",
```

Comment:
Do we really need these in the profile?

Reply:
because of the core.extension.yml, yeah. the tests dont work otherwise

Reply:
These aren't needed by the profile but rather by the stack. We could probably pull them out of core.extension couldn't we?

Reply:
we'd have to move them to the environment splits but then i'm guessing it would be in every config split. i dont think we can get around it. unless we move them to `dev-dependencies` here.

Reply:
It doesn't sound worth the move. They don't really hurt being in here so let's not fight this fight until later. We will have the need for individual sites to have different sets of modules enabled but that will be tackled in our update practices.